modules/@apostrophecms/http/lib/big-upload-middleware.js

```diff
@@ -163,7 +163,7 @@ module.exports = (self) => ({
         });
       }
       let n = 0;
-      req.files = {};
+      req.files = [];
```

Comment:
The interface big-upload-middleware provides must not change to match multer's interface because we have already documented that it matches multiparty's interface. You can still use multer to implement it, but the `req.files` object exposed at the end must behave like the multiparty one to avoid a backwards compatibility break.